Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py

```diff
@@ -261,19 +261,24 @@ def edl_update_from_external_file(list_name: str, file_path: str, type_: str):
     list_data = dict_of_lists.get(list_name, None)
     file_data = edl_get_external_file(file_path)
 
-    set_internal = set(list_data)
-    set_external = set(file_data.split('\n'))
-    set_external.discard('')
-
-    if type_ == 'merge':
-        unified = set_internal.union(set_external)
-        list_data_new = list(unified)
-    else:  # type_ == 'override'
-        list_data_new = list(set_external)
-
-    dict_of_lists.update({list_name: list_data_new})
-
-    return list_data_new
+    if list_data:
+        set_internal = set(list_data)
+        set_external = set(file_data.split('\n'))
+        set_external.discard('')
+        if type_ == 'merge':
+            unified = set_internal.union(set_external)
+            list_data_new = list(unified)
+        else:  # type_ == 'override'
+            list_data_new = list(set_external)
+        dict_of_lists.update({list_name: list_data_new})
+        demisto.setIntegrationContext(dict_of_lists)
+        return list_data_new
+    else:
+        demisto.log(str(file_data))
```

Comment:
what is the size we are expecting to log here?

Reply:
i should remove this line.

Reply:
@bakatzir take a look on the next line with the log